File: server/xxx.py
#!/usr/bin/python
import os, sqlite3, time
from serial.serialutil import to_bytes

conn = sqlite3.connect('/home/pi/workspace/raspi_net/server/db/sys.db')
c = conn.cursor()
print "Opened database successfully";

cursor = c.execute("SELECT TEMP, HUMI FROM TEMP_HUMI ORDER BY ID DESC LIMIT 1")
row = cursor.fetchone()
command = str(round(row[0],2)) + "_" + str(round(row[1],2)) + "_" + str(time.strftime("%H", time.localtime()))

print "Operation done successfully";
conn.close()

print command + "\n";

d = to_bytes(command)
serial = os.open('/dev/ttyACM0', os.O_RDWR | os.O_NOCTTY | os.O_NONBLOCK)
os.write(serial, d)

File: server/xxx.cpp
#include <stdio.h>
#include <stdlib.h>
#include <time.h>
#include <sqlite3.h>
#include <sys/types.h>
#include <sys/stat.h>
#include <fcntl.h>
#include <unistd.h>


int main(int argc, char* argv[])
{
  int fd;
  char command[] = "18_100_10";
  fd = open("/dev/ttyACM0", O_RDWR | O_NOCTTY | O_NONBLOCK);
  while(true){
  sleep(2);
  write(fd, command, sizeof(command));
  sleep(6);
  }
}
